### ISSUE TITLE
[Feature Request] Add a way to customize BaseUiElementFactory or just export to

### ISSUE THREAD

**Issue by yelhouti** [mebjas/html5-qrcode#799]
**Is your feature request related to a problem? Please describe.**
I often want to add classes to the elements created by the `BaseUiElementFactory`. Insteading of rewriting the css to match the classes defined.

**Describe the solution you'd like**
Either export in index.js the `BaseUiElementFactory` and I will overwrite it.
Or even better, add a new param to `Html5QrcodeScanner` called `UiElementFactory` which would allow us to extends the `BaseUiElementFactory` and provide an instance of the extend class to be used. (also use instance method instead of static `createElement`)

**Comment by sulimanbenhalim** [comment]
I totally agree with this feature request! Being able to extend `BaseUiElementFactory` would offer much-needed flexibility for custom styling and behaviors without having to override the existing styles.

Having a `UiElementFactory` parameter in `Html5QrcodeScanner` would make it much easier to customize the UI elements, especially in complex projects where the default UI might not fit the overall design scheme.

This would be an invaluable addition to the library. Is there any way we can help to get this feature implemented?
